aws_lambda_builders/workflows/nodejs_npm/npm.py

```diff
@@ -0,0 +1,53 @@
+"""
+Wrapper around calling npm through a subprocess.
+"""
+
+import logging
+
+from aws_lambda_builders.workflows.python_pip.utils import OSUtils
+
+LOG = logging.getLogger(__name__)
+
+
+class NpmError(Exception):
+    pass
+
+
+class NpmExecutionError(NpmError):
+    def __init__(self, err):
+        super(NpmExecutionError, self).__init__(
+            'NPM failed: %s' % err)
+
+
+class SubprocessNpm(object):
+
+    def __init__(self, osutils=None, npm_exe=None):
+        if osutils is None:
+            osutils = OSUtils()
+        self.osutils = osutils
+
+        if npm_exe is None:
+            npm_exe = 'npm'
+
+        self.npm_exe = npm_exe
+
+    def main(self, args, cwd=None):
```

Comment:
Actually, its a bit weird to call this `main()`. How about `run()` instead?

Reply:
resolved in https://github.com/gojko/aws-lambda-builders/commit/3e945b281e63e82d83e0fe3fe8f71b9c50f978aa